linux-custom.pkr.js/linux-custom.pkr.js

```diff
@@ -0,0 +1,18 @@
+{
+  "variables": { },
+  "builders": [
+    {
+      "type": "docker",
+      "image": "tipibuild/tipi-linux-custom:{{cmake_re_source_hash}}",
```

Comment:
same here let's just name it linux-custom

Reply:
done